draft-ietf-tls-esni.md

```diff
@@ -926,6 +933,9 @@ a compliant ECH application MUST implement the following HPKE cipher suite:
 
 ## Security and Privacy Goals {#goals}
 
+[[TODO: Modify the text below to account for the acceptance confirmation signal
+being derived from the handshake secret.]]
```

Comment:
The relevant text here notes that active attackers can distinguish real from fake ECH, which, as written, is probably still true with this change. (It could probe for the outer name, and then memcmp that to what's in a packet, for example.) So I might drop this TODO and address overhauling this section as part of #354.

Reply:
Done.